woodwork/tests/test_utils.py

```diff
@@ -164,11 +164,47 @@ def test_list_semantic_tags():
                 assert name in log_type.standard_tags
 
 
-def test_read_csv_no_params(sample_df_pandas, tmpdir):
+def test_read_file_errors_no_content_type(sample_df_pandas, tmpdir):
+    filepath = os.path.join(tmpdir, 'sample')
+    sample_df_pandas.to_csv(filepath, index=False)
+
+    no_type_error = "Content type could not be inferred. Please specify content_type and try again."
+    with pytest.raises(RuntimeError, match=no_type_error):
+        ww.read_file(filepath=filepath)
+
+
+def test_read_file_errors_unhandled_content_type(sample_df_pandas, tmpdir):
+    filepath = os.path.join(tmpdir, 'sample')
+    sample_df_pandas.to_csv(filepath, index=False)
+
+    no_type_error = "Content type could not be inferred. Please specify content_type and try again."
+    with pytest.raises(RuntimeError, match=no_type_error):
+        ww.read_file(filepath=filepath)
```

Comment:
Same as `test_read_file_errors_no_content_type` above?

Reply:
renamed and changed to `test_read_file_errors_unsupported`